extism_test.go

```diff
@@ -701,6 +703,55 @@ func TestJsonManifest(t *testing.T) {
 	}
 }
 
+func TestObserve(t *testing.T) {
```

Comment:
Maybe either in this test, or in a separate test, could we add a scenario where multiple plugins are instantiated and run, but share the same adapter?

Reply:
good idea